plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -19,6 +19,17 @@ deploy.preferences.dialog.flex.pricing=There is no free quota for the App Engine
  pricing information.
 build.error.dialog.title=Build Error in Project
 build.error.dialog.message=Project has error(s). Try again after fixing the problem.
+vm.is.jre.title=Deploying JSPs requires a JDK
+vm.is.jre.proceed=\
+ Project "{0}" is configured to use:\n\
+ \n\
+ \tJava System Library: {1}\n\
+ \tLocation: {2}\n\
+ \n\
+ This Java System Library appears to be a Java Runtime Environment (JRE). \
```

Comment:
perhaps "This appears to be a Java Runtime Environment (JRE)." or even
 "This is a Java Runtime Environment (JRE)."

Reply:
Since it's a heuristic, I'll keep the _appears_.